packages/react/package.json

```diff
@@ -22,23 +22,31 @@
     "build:cjs": "tsup src/index.tsx --format cjs --dts",
     "dev": "tsup --watch && cp src/compiled.css dist",
     "clean": "",
-    "test": "vitest",
-    "lint": "eslint \"**/*.ts*\""
+    "lint": "eslint \"**/*.ts*\"",
+    "test": "vitest --watch false",
+    "test:watch": "vitest"
   },
   "dependencies": {
     "react": "^18.2.0"
   },
   "devDependencies": {
+    "@tanstack/react-query": "^5.29.2",
```

Comment:
Same, there might be good value stepping away from loose dependencies and it might prevent from possible issues